src/activy/app_update.py

```
@@ -10,61 +10,61 @@ def update_stage1(device, templates):
-    Branching: If an update is needed, 101 is initiated
+    Branching: If an update is needed, 1_1 is initiated
     """
     try:
-        tryCheckStage(device, "1o101", templates)
+        tryCheckStage(device, "1To1_1", templates)
         print("Stage 1 completed")
-        update_stage101(device, templates)
+        update_stage1_1(device, templates)
         return True
-    except:
+    except ValueError as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)